docs/tutorials/notebooks/sparse_monge_displacements.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>Maybe mention the time complexity?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>it's O(kd) :) </p>